etc/scripts/json2csv.py

```diff
@@ -106,239 +75,148 @@ def load_scan(json_input):
-        """
-        {
-          "type": "Nuget",
-          "packaging": "archive",
-          "primary_language": null
-        }
-        """
-        for info in entry.get('packages', []):
-            inf = OrderedDict(Resource=location)
-            inf.update(('package_' + k, v or '') for k, v in info.items())
-            keys.update(inf.keys())
-            rows.append(inf)
-
-        for info in [promote_inner_keys(y) for x in entry.get('package_details', []) for y in flatten(x)]:
-            inf = OrderedDict(Resource=location)
-            inf.update(('package_details_' + k, v or '') for k, v in info.items())
-            keys.update(inf.keys())
-            rows.append(inf)
-
-        """
-        {
-          "url": "http://nexb.com/",
-          "start_line": 3,
-          "end_line": 3
-        },
-        """
-        for url in entry.get('urls', []):
-            url = OrderedDict(Resource=location)
-            for k, val in url.items():
-                url[k] = val
-                keys.add(key)
-            rows.append(url)
-
-        """
-        {
-          "email": "info@winimage.com",
-          "start_line": 1124,
-          "end_line": 1124
-        }
-        """
-        for email in entry.get('emails', []):
-            email = OrderedDict(Resource=location)
+            for key, header in (('statements', 'copyright'), ('holders', 'copyright_holder'), ('authors', 'author')):
+                for cop in copy_info.get(key, []):
+                    inf = OrderedDict(Resource=path)
+                    inf[header] = cop
+                    inf['start_line'] = start_line
+                    inf['end_line'] = end_line
+                    yield inf
+
+        for email in scanned_file.get('emails', []):
+            email_info = OrderedDict(Resource=path)
             for k, val in email.items():
-                email[k] = val
-                keys.add(key)
-            rows.append(email)
-
-
-    # keys = ['Resource'] + [x for x in keys if x != 'Resource']
-    all_rows = []
-    for row in rows:
-        d = OrderedDict()
-        for k in keys.keys():
-            d[k] = row.get(k, '')
-        all_rows.append(d)
+                email_info[k] = val
+            yield email_info
 
-    return all_rows
+        for url in scanned_file.get('urls', []):
+            url_info = OrderedDict(Resource=path)
+            for k, val in url.items():
+                url_info[k] = val
+            yield url_info
+
+        excluded_columns = ('packaging',
+                            'payload_type',
+                            'keywords_doc_url',
+                            'download_sha1',
+                            'download_sha256',
+                            'download_md5',
+                            'code_view_url',
+                            'vcs_tool',
+                            'vcs_revision',
+                            'license_expression')
+
+        for package in scanned_file.get('packages', []):
```

Comment:
Can you make sure that for the package version field you always pad it with `v space` if present .... otherwise loading a version in a spreadsheet like Excel or LibreOffice will mistake a `1.10` version for `1.1` when loading?